inspector/src/modules/log_module.ts

```diff
@@ -210,6 +231,9 @@ export default function LogModule({
   onDestroyFns.push(
     logView.subscribe(STATE_PROPS.LOGS_HISTORY, onLogsHistoryChange, true)
   );
+  onDestroyFns.push(
+    configState.subscribe(STATE_PROPS.TIME_REPRESENTATION, reloadLogsHistory, true)
+  )
```

Comment:
To remove and just call `    onLogsHistoryChange(UPDATE_TYPE.REPLACE, logView.getCurrentState(STATE_PROPS.LOGS_HISTORY))` at the end of `toggleTimeRepresentation`?